Userland/Libraries/LibGfx/BMPLoader.cpp

```diff
@@ -1098,7 +1095,7 @@ static bool uncompress_bmp_rle_data(BMPLoadingContext& context, ByteBuffer& buff
         }
         case RLEState::Meta:
             if (!streamer.has_u8())
-                return false;
+                return Error::from_string_literal("Could not read 8 bits");
```

Comment:
Let's be more consistent with the error messages. In some places you have "Could not read 8 bits", in others you have "Cannot read 8 bits". Let's stick with the latter, because this is happening before we try to read (whereas the former sounds like we attempted reading then failed).